packages/builder/src/pages/builder/portal/settings/ai/logos/Anthropic.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  export let height
+  export let width
+</script>
+
+<svg
+  id="katman_1"
```

Comment:
Do you need this `id` and `data-name`? They don't appear to be used within the SVG.